docs-ref-conceptual/authenticate-azure-cli.md

```diff
@@ -142,8 +140,8 @@ az login
 ```
 
 At the current stage of development, there are a few known limitations to WAM:
-_ WAM is available on Windows 10   and later, and on Windows Server 2019 and later. On Mac , Linux, and earlier versions of Windows, we automatically fall back to a browser.  
-_ Microsoft Accounts (for example @outlook.com or @live.com) are not supported for the time being  . We are working with the Microsoft  Identity team to bring the support in the near future.
+_ WAM is available on Windows 10   and later, and on Windows Server 2019 and later. On Mac, Linux, and earlier versions of Windows, we automatically fall back to a browser.  
+_ Microsoft Accounts (for example @outlook.com or @live.com) aren't supported for the time being. We're working with the Microsoft  Identity team to bring the support soon.
```

Comment:
replace `in the near future` with `later`.

remove extra space between `Microsoft` and `Identity`

Reply:
Revised